added doc for force flag (#5881)
DataDog/documentation: content/en/api/service_level_objectives/service_level_objectives_delete.md

Permanently delete a SLO.

> If an SLO is used in a dashboard, the DELETE /v1/slo/ endpoint will return a 409 conflict error because the SLO is referenced in a dashboard.

Reviewer: ```suggestion
If an SLO is used in a dashboard, the DELETE /v1/slo/ endpoint returns a 409 conflict error because the SLO is referenced in a dashboard.
```

**ARGUMENTS**:

> * **`force`** [*optional*]:

Reviewer: ```suggestion
* **`force`** [*optional*, *default*=**False**]:
```

>     Boolean: Force delete the SLO. The SLO will be deleted regardless of if it's referenced in a dashboard. Defaultfalse.

Reviewer: ```suggestion
    Boolean: Force delete the SLO. The SLO is deleted even if it's referenced in a dashboard.
```

**ARGUMENTS**:

* **`force`** [*optional*]:

>     Boolean: Force delete SLO.  Required if SLO exists in a dashboard.  Default `false`

Reviewer 2: Could you reword this as: 

> Permanently delete an SLO. 
>
>  If an SLO is used in a dashboard, the `DELETE /v1/slo/` endpoint will return a `409` conflict error because the SLO is referenced in a dashboard.
>
> **ARGUMENTS**:
>
> * **`force`** [*optional*]:
>
> Boolean: Force delete the SLO. The SLO will be deleted regardless of if it's referenced in a dashboard.  Default`false`.

Author: @armcburney nice additions.  Just added 👍
